CMakeLists.txt

```diff
@@ -51,12 +51,12 @@ find_package(OpenCV 3 REQUIRED)
 find_package(Doxygen)
 
 ### for python 2 use this:
-find_package(Boost COMPONENTS system python-py27 REQUIRED)
-find_package(PythonLibs 2.7 EXACT REQUIRED) 
+#find_package(Boost COMPONENTS system python-py27 REQUIRED)
+#find_package(PythonLibs 2.7 EXACT REQUIRED) 
 
 ### for python 3 use this:
-#find_package(Boost COMPONENTS system python-py35 REQUIRED)
-#find_package(PythonLibs 3 EXACT REQUIRED) 
+find_package(Boost COMPONENTS system python3 REQUIRED)
```

Comment:
I tested this with the boost-python package provided by ubuntu.  The **python3** component is not found (same for python2).  I think this is due to the different naming followed by the ubuntu build. 

In order to keep the compatibility with the default boost package there should be a check here. Otherwise accepting this fix would break the build process for everyone else.